### ISSUE TITLE
propose-downstream: inform user about errors

### ISSUE THREAD

**Issue by TomasTomecek** [packit-service/packit-service#71]
Especially when packit service doesn't have perms to push to the repo, it should create a new issue on the repo and tell the user.

Current error message in logs:
```
packit.exceptions.PackitException: We were unable to push to dist-git: [remote rejected] (pre-receive hook declined)
```
The pre-receive hook declines when we don't have the perms.

**Comment by TomasTomecek** [comment]
Once we have logs in packit-service, we can set commit check status to failed and link to the logs.

**Comment by dhodovsk** [comment]
Requires: https://github.com/packit-service/packit-service/issues/1

**Comment by lachmanfrantisek** [comment]
Currently, we are creating an issue when the `propose-downstream` fail:

https://github.com/packit-service/packit-service/blob/824783842994278625d8be26873af4271cdc4548/packit_service/worker/github_handlers.py#L228

And in the issue, you can retrigger the process.

See the related PR for the screenshots: https://github.com/packit-service/packit-service/pull/300

- [ ] The missing piece is the commit-status (ideally with the link to the logs).

**Comment by TomasTomecek** [comment]
I'd say the issue is good enough for now, so we can close this. Setting the commit status sounds like an overkill to me.

**Comment by jpopelka** [comment]
Not sure what is "commit status", but I also think it's OK as it is now (e.g. https://github.com/packit-service/ogr/issues/348)
